1. CWE Base Mod/Cold War Era Mod (CWE)/common/country_definitions/00_south_east_asia.txt

```diff
@@ -562,4 +562,16 @@ BOU = { # Bougainville
 	
 	cultures = { melanesian }
 	capital = STATE_BOUGAINVILLE
+}
+
+# FRENCH INDO CHINA
+FIC = {
+	color = { 20  50  210 }
+
+	country_type = recognized
+
+	tier = colony
```

Comment:
Should use the empire rage if the constituent states are kingdom, if not then kingdom if the constituent states are principalities

Reply:
Empire then, Cambodia etc were protectorate kingdoms